CockpitPlugin/src/main/java/com/polidea/cockpitplugin/generator/BaseCockpitGenerator.kt

```diff
@@ -64,21 +64,21 @@ abstract class BaseCockpitGenerator {
     }
 
     protected fun mapToTypeClass(param: Param<*>): Class<*> {
-        return when (param) {
```

Comment:
`when` expressions is not needed any more. We can simplify it to `param.value::class.java`